### ISSUE TITLE
Nessus XML import - duplicate entries in report

### ISSUE THREAD

**Issue by ITGSean** [SerpicoProject/Serpico#330]
Hi guys

I'm not sure if this is a bug or a feature! (This is with auto-import enabled)

I've just noticed with the latest version (git pulled a few hours ago), that if you use the Nessus XML import into a report, it brings in mapped findings from the findings DB, but also imports the original finding from the XML file.

This results in duplicate findings in the report, which at the moment, if you have a lot of findings, are a pain to delete.

So, two things:

1)  Can a selection box be added to lists to enable multi-selection of list items for bulk operations like deletion?
2)  Can there be the option to import findings with no valid mappings and include findings from the main findings DB which have a valid mapping, but not both at the same time?

I hope the above is clear.  Makes _some_ sense to me anyway :)

Cheers,

Sean

**Comment by BuffaloWill** [comment]
Thanks for the submission. Do you have a nessus scan we could use to recreate the issue? A scan of a bug bounty site is fine.

**Comment by ITGSean** [comment]
@BuffaloWill here you go:

[alogolia.zip](https://github.com/SerpicoProject/Serpico/files/1269872/alogolia.zip)

**Comment by ITGSean** [comment]
Hi guys.
This is still an issue.

I've also noticed that the different imports don't work in consistent ways.

Nessus XML import will include the nessus plugin output in the notes field in the nessus finding, but no output is recorded in a mapped finding, making it necessary to copy/paste either from the imported duplicate finding, or from the nessus console.

Using a MSF RPC import doesn't record any output in the mapped finding either.

Burp findings just import the hostname and not any affected URI's

In all cases for nessus import, the port number is never imported, making it a laborious task to add them in when a lot of hosts are affected.

Hope this helps move things along.

Cheers

Sean

**Comment by ITGSean** [comment]
Good way to get rid of an issue, close both duplicates!

You've got a cyclical "closing as a duplicate of" each issue here @BuffaloWill

**Comment by BuffaloWill** [comment]
@ITGSean, this issue is still open. What are you referring to?

**Comment by BuffaloWill** [comment]
@ITGSean. I started working on this issue but ran into a road block (thanks for the import file btw). 

These were the steps I followed to recreate:
1. Enabled "Auto Import" and "Nessusmap" in the configuration, restart.
2. Assigned plugin-id 56818 to Cross Site Scripting in the Findings Database.
3. Imported the Nessus data from the issue

No duplicate findings were added. To confirm the issue, I should be seeing duplicate findings, correct?

Also, (you are probably aware) but bulk deletion was added.
